src/get-started.md.njk

```diff
@@ -6,7 +6,7 @@ layout: layout-single-page-prose.njk
 
 {% from "_example.njk" import example %}
 
-This guide explains how to create prototypes using the GOV.UK Design System and GOV.UK Prototype Kit.
+This guide explains how to create prototypes using the GOV.UK Design System and GOV.UK Prototype Kit. For information on how to use production code visit the [GOV.UK Frontend](https://github.com/alphagov/govuk-frontend).
```

Comment:
After discussing this with @dashouse - suggest changing:

"For information on how to use production code visit the [GOV.UK Frontend](https://github.com/alphagov/govuk-frontend)."

to:

"If you want to use the Design System in production, you’ll need to include GOV.UK Frontend in your project.

Read guidance on how to install [GOV.UK Frontend on GitHub](https://github.com/alphagov/govuk-frontend).